obs_example/Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM opensuse:latest
+
+# Make obs_install available
+ADD obs_install /usr/bin/obs_install
```

Comment:
All instances of `obs_install` in this file need to be replaced by `obs_pkg_mgr`